apps/rule-manager/client/src/ts/components/PageNotFound.tsx

```diff
@@ -0,0 +1,77 @@
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiTitle } from "@elastic/eui"
+import { css } from "@emotion/react"
+import { Link } from "react-router-dom"
+import typey from "../../images/typey.gif"
+import typeyArrow from "../../images/typey-arrow.gif"
+import styled from "@emotion/styled"
+
+const TypeyContainer = styled.div`
+    display: flex;
+    flex-direction: horizontal;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1.1rem;
+    image-rendering; pixelated
+    font-smooth: never;
+    -webkit-font-smoothing : none;
+    line-height: 1.6rem;
+    position: relative;
+    margin-top: 20px;
+`;
+
+const TypeySpeechBubble = styled.div`
+    position: relative;
+    background-color: #FCFECC;
+    display: block;
+    height: fit-content;
+    padding: 8px;
+    border: 1px solid black;
+    border-radius: 8px;
+    margin-left: 8px;
+`;
+
+const TypeyButton = styled.button`
+    font-family: Arial, Helvetica, sans-serif;
+    border: 1px solid #C3C59E;
+    padding: 3px 10px;
+    border-radius: 4px;
+    :hover {
+        background-color: #f3f5c1;
+    }
+    margin-top: 4px;
+`;
+
+const TypeyText = styled.p`
+    color: black;
+`
+
+const TypeyArrow = styled.img`
+    position: absolute;
+    left: -13px;
+    bottom: 11px;
+    background: none;
+    content: "";
+    transform: scaleY(-1) rotate(90deg);
+}`
```

Comment:
This `}` looks a little rogue

Reply:
The lack of syntax highlighting in those emotion css strings has revealed my inability to type

Reply:
The only reason I saw these was because IntelliJ highlighted them for me, so maybe there's something you can do with your local setup? 🤷